safe_transaction_service/history/indexers/tx_processor.py

```diff
@@ -82,9 +83,14 @@ class SafeTxProcessor(TxProcessor):
     Processor for txs on Safe Contracts v0.0.1 - v1.0.0
     """
 
-    def __init__(self, ethereum_client: EthereumClient):
+    def __init__(
+        self,
+        ethereum_client: EthereumClient,
+        ethereum_tracing_client: Optional[EthereumClient],
```

Comment:
Some docs documenting what happens if it is not set would be nice

Reply:
Updated in a the new commit